doc/Extensions/Applications.md

```diff
@@ -115,6 +116,27 @@ snmpwalk <various options depending on your setup> localhost NET-SNMP-EXTEND-MIB
 (If you get error like "Can't locate LWP/Simple.pm". libwww-perl needs to be installed: apt-get install libwww-perl)
 2. On the device page in Librenms, edit your host and check the `Apache` under the Applications tab.
 
+### Asterisk
+A small shell script that reports various Asterisk call status.
+
+##### SNMP Extend
+1. Copy the [asterisk script](https://github.com/librenms/librenms-agent/blob/master/snmp//asterisk) to `/etc/snmp/` on your asterisk server.
```

Comment:
`snmp//asterisk`
There is a double character /

Reply:
Fixed!